pyscipopt/expr.pxi

```diff
@@ -135,7 +132,12 @@ cdef class Expr:
         '''turn it into a constraint'''
         return _expr_richcmp(self, other, op)
 
+    def normalize(self):
+        '''remove terms with coefficient of 0'''
+        self.terms =  {t:c for (t,c) in self.terms.items() if c != 0.0}
+
     def __repr__(self):
+        self.normalize()
```

Comment:
Maybe this should be reverted, too, since it's redundant now and I don't like side-effects :-)